mitmproxy/addons/dumper.py

```diff
@@ -124,6 +125,9 @@ def _echo_request_line(self, flow):
             url = flow.request.pretty_url
         else:
             url = flow.request.url
+        terminalWidthLimit = shutil.get_terminal_size()[0] - 25
```

Comment:
Rare case, but what happens if `get_terminal_size()[0]` is < 25? Why do we substract 25 here at all?